user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/FAQ/Dashboards_and_Low_Code_Apps_FAQ.md

```diff
@@ -19,3 +19,13 @@ To resolve these:
 - Verify that no firewall or proxy is interfering with the setup of the WebSocket connection.
 
 - Use a modern browser that supports WebSockets. The DataMiner web apps do not support Internet Explorer.
+
+## What should I do if I don't see my protocol changes applied in a GQI query result?
+
+If you've made protocol changes but don't see them reflected in your GQI query results, it's likely due to GQI caching certain protocols for faster performance.
+
+To resolve this:
+1. Launch your system's Task Manager utility.
+2. Look through the list of processes for the *Skyline DataMiner Helper* process. This process is responsible for managing GQI.
```

Comment:
1,2,3 are not encouraged to use. Use 1. everywhere: https://docs.dataminer.services/CONTRIBUTING.html#lists